stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -184,6 +184,50 @@ function eigen!(A::AbstractMatrix, C::Cholesky; sortby::Union{Function,Nothing}=
     GeneralizedEigen(sorteig!(vals, vecs, sortby)...)
 end
 
+# Bunch-Kaufmann (LDLT) based solution for generalized eigenvalues and eigenvectors
+function eigen(A::AbstractMatrix{T}, B::BunchKaufman{T,<:StridedMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    eigen!(copy(A), copy(B); sortby)
+end
+function eigen!(A::AbstractMatrix{T}, B::BunchKaufman{T,<:StridedMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
```

Comment:
I believe that whenever we have direct calls to `LAPACK`, we restrict the argument type to `StridedMatrix`, most likely together with eltype restrictions like `BlasFloat` or `BlasReal` or something.

Reply:
Thank you for the comment, I agree. I've incorporated both the `StridedMatrix` and `BlasFloat` changes in the following commit: https://github.com/aravindh-krishnamoorthy/julia/commit/8b8f556d7095452e14a3bc7c46c4bc2f34d6bb9d.